src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,127 @@
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        {Object.keys(GenderFilters).map(filter => {
+          const filterKey = GenderFilters[filter as keyof typeof GenderFilters];
+
+          return (
+            <Link
+              key={filterKey}
+              className={classNames({
+                'is-active': searchParams.get(SearchQueries.sex) === filterKey,
+              })}
+              to={{
+                pathname: '/people',
+                search: getSearchWith(searchParams, {
+                  [SearchQueries.sex]: filterKey,
+                }),
+              }}
+            >
+              {filter
+                .split('')
+                .map((ch, i) => (i == 0 ? ch.toLocaleUpperCase() : ch))}
+            </Link>
+          );
+        })}
+      </p>
+
+      <div className="panel-block">
+        <p className="control has-icons-left">
+          <input
+            data-cy="NameFilter"
+            type="search"
+            className="input"
+            placeholder="Search"
+            onChange={e => {
+              setSearchParams(
+                getSearchWith(searchParams, {
+                  [SearchQueries.query]: e.target.value || null,
+                }),
+              );
+            }}
+          />
+
+          <span className="icon is-left">
+            <i className="fas fa-search" aria-hidden="true" />
+          </span>
+        </p>
+      </div>
+
+      <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            {['16', '17', '18', '19', '20'].map(centr => {
+              return (
+                <Link
+                  key={centr}
+                  data-cy="century"
+                  className={classNames('button mr-1', {
+                    'is-info': searchParams.getAll('centuries').includes(centr),
+                  })}
+                  to={{
+                    pathname: '/people',
+                    search: toggleCentruies(centr),
+                  }}
+                >
+                  {centr}
+                </Link>
+              );
+            })}
+          </div>
+
+          <div className="level-right ml-4">
+            <Link
+              data-cy="centuryALL"
+              className={classNames('button is-success', {
+                'is-outlined': searchParams.get('centuries'),
+              })}
+              to={{
+                pathname: '/people',
+                search: getSearchWith(searchParams, {
+                  centuries: null,
+                }),
+              }}
+            >
+              All
+            </Link>
+          </div>
+        </div>
+      </div>
+
+      <div className="panel-block">
+        <Link
+          className="button is-link is-outlined is-fullwidth"
+          to={{
+            pathname: '/people',
+            search: getSearchWith(searchParams, {
+              centuries: [],
```

Comment:
Code Quality Issue: Avoid using magic strings. Here 'centuries', 'sex', and 'query' are magic strings. It would be better to create constants for them with descriptive names, so the code is more readable and maintainable.